src/editing_tools/AdjustTool.vala

```diff
@@ -134,6 +132,7 @@ public class EditingTools.AdjustTool : EditingTool {
             var button_box = new Gtk.ButtonBox (Gtk.Orientation.HORIZONTAL) {
                 margin_top = 12
             };
+
```

Comment:
Nitpicking but this has a trailing `;` we don't put a newline here IMO.

Have a newline:

```
if (i < 10) {
    text = "";
}

var grid = new Gtk.Grid ();
…
```

No newline:

```
var grid = new Gtk.Grid () {
    margin = 6
};
grid.add (label);
…
```